modules/drivers/druid/test/metabase/driver/druid_test.clj

```diff
@@ -202,13 +202,13 @@
 
 ;; count-where
 (expect-with-driver :druid
-  [[17.0]]
+  [[951]]
```

Comment:
@sbelak I ran some queries locally and can confirm 951 is the right answer. 

How did we merge tests for the wrong answer (`17`) in the first place?

Reply:
It's diffed off of sum-where branch ... which at some point had the wrong result

Reply:
ok